src/cbmc/bmc_util.h

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************\
+
+Module: Bounded Model Checking Utilities
+
+Author: Daniel Kroening, Peter Schrammel
+
+\*******************************************************************/
+
+/// \file
+/// Bounded Model Checking Utilities
+
+#ifndef CPROVER_CBMC_BMC_UTIL_H
+#define CPROVER_CBMC_BMC_UTIL_H
+
+#include <memory>
+
+#include <goto-programs/safety_checker.h>
+
+class goto_tracet;
+class memory_model_baset;
+class message_handlert;
+class namespacet;
+class optionst;
+class prop_convt;
+class symex_bmct;
+class symex_target_equationt;
+struct trace_optionst;
+class ui_message_handlert;
+
+void convert_symex_target_equation(
+  symex_target_equationt &,
+  prop_convt &,
+  message_handlert &);
+
+void report_failure(ui_message_handlert &);
```

Comment:
All these methods now have names that don't specify what they do that clearly, because they're no longer in `bmct`. Move them into a namespace `bmc_utilt`?